mod/projects/spec/set/self/project_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe Card::Set::Self::Project do
+  specify "view core" do
+    expect_view(:core).to have_tag("div.card-slot") do
```

Comment:
I don't understand why this passes.  Why does the outer div have a card-slot?

Reply:
OK, now I understand how this passes.

It looked to me like this was testing to see whether there was a .card-slot with a .tabbable and a .tab-content inside it.  There isn't, but that's not what this is testing.   To test that, the 2nd and 3rd `have_tag` would need to be `with_tag`.

As written, the test is just testing whether each of those three divs exist at all, and they do (just not inside each other)

If you just want to test that they exist, try:
```
  expect_view(:core).to have_tag("div.card-slot") 
                                  .and have_tag("div.tabbable")
                                  .and have_tag("div.tab-content")
```

... or, if you want to look for a div that is outside all three of them, you need to add a class to the outer div in the haml.

In either case, I think it would be wise to make sure that the test confirms that we are seeing the featured projects card.  (and I assume you've confirmed in the browser that this all looks right)